.github/workflows/auto-bump-up.yml

```diff
@@ -33,9 +33,9 @@ jobs:
       - name: Set each submodule version to env
         if: env.NEW_TAG
         run: |
-          FINSCHIA_SDK_VERSION=$(grep 'github.com/Finschia/finschia-sdk' go.mod | awk '{print $2}' | awk -F '-' '{if ($3 != "") {print $3} else {print $1"-"$2} }')
+          FINSCHIA_SDK_VERSION=$(grep 'github.com/Finschia/finschia-sdk' go.mod | awk '{print $2}' | awk -F '-' '{if ($3 != "" && $2 != "") {print $3} else  if ($3 != "") {print $1"-"$2} else {print $1} }')
```

Comment:
it seems the second option is wrong. Do you wanna include `rc` version for that case?
Then, it should be `($3 == "")`

Reply:
In this case `v0.48.0-rc2.0.20230905020121-022614f80a0d`, env should be set `022614f80a0d`.
But case like `v0.48.0-rc2`, env should be set `v0.48.0-rc2`.
So I intend it.